### ISSUE TITLE
Command palette demo not working in Kitty

### ISSUE THREAD

**Issue by Darkstarinternet** [Textualize/textual#3392]
Have you checked closed issues? https://github.com/Textualize/textual/issues?q=is%3Aissue+is%3Aclosed
Yes

Please give a brief but clear explanation of the issue. If you can, include a complete working example that demonstrates the bug. **Check it can run without modifications.**

I'm trying to run the command palette example provided in the documentation. When I run it in either the default macOS terminal it works and it also works when run in Alacrity or the PyCharm terminal, however when I press CTRL+\ when using the Kitty terminal nothing happens. Checking the Textual console doesn't display any of the CommandPalette output I get when using the other terminal emulators so the problem appears to be specific to Kitty. I am using Kitty 0.30.0 and have tried starting it with default config but still get the problem.

It will be helpful if you run the following command and paste the results:

```
textual diagnose
```

# Textual Diagnostics

## Versions

| Name    | Value  |
|---------|--------|
| Textual | 0.38.1 |
| Rich    | 13.5.3 |

## Python

| Name           | Value                                                                                  |
|----------------|----------------------------------------------------------------------------------------|
| Version        | 3.11.5                                                                                 |
| Implementation | CPython                                                                                |
| Compiler       | Clang 14.0.3 (clang-1403.0.22.14.1)                                                    |
| Executable     | /Users/Tom/Library/Caches/pypoetry/virtualenvs/python-demos-0pX9poEm-py3.11/bin/python |

## Operating System

| Name    | Value                                                                                            |
|---------|--------------------------------------------------------------------------------------------------|
| System  | Darwin                                                                                           |
| Release | 22.6.0                                                                                           |
| Version | Darwin Kernel Version 22.6.0: Wed Jul  5 22:21:56 PDT 2023; root:xnu-8796.141.3~6/RELEASE_X86_64 |

## Terminal

| Name                 | Value       |
|----------------------|-------------|
| Terminal Application | Kitty       |
| TERM                 | xterm-kitty |
| COLORTERM            | truecolor   |
| FORCE_COLOR          | *Not set*   |
| NO_COLOR             | *Not set*   |

## Rich Console options

| Name           | Value                |
|----------------|----------------------|
| size           | width=192, height=45 |
| legacy_windows | False                |
| min_width      | 1                    |
| max_width      | 192                  |
| is_terminal    | True                 |
| encoding       | utf-8                |
| max_height     | 45                   |
| justify        | None                 |
| overflow       | None                 |
| no_wrap        | False                |
| highlight      | None                 |
| markup         | None                 |
| height         | None                 |

Feel free to add screenshots and / or videos. These can be very helpful!

**Comment by willmcgugan** [comment]
Seems to work here. If you run `textual keys` do you see any output when you hit ctrl+backslash ?

What version of Kitty are you on?

**Comment by davep** [comment]
As with Will, I'm running it in kitty (0.30.0 here) just fine; in fact kitty is my daily driver and it's what the command palette was mainly tested and developed in.

**Comment by willmcgugan** [comment]
I had an older version. Upgraded to 0.30.0 and everything works fine. @Darkstarinternet have you changed any settings re key bindings maybe? In Kitty or the OS settings.

**Comment by Darkstarinternet** [comment]
`textual keys` gives the following when pressing `CTRL+\` which is clearly wrong:

Key(key='grave_accent', character='`', name='grave_accent', is_printable=True)

This looks like a problem with my kitty set up, I'll investigate further but it doesn't look like a textual problem, thanks for the help guys, I'm loving textual!

**Comment by davep** [comment]
@Darkstarinternet Do let us know if you get to the bottom of it; it will be useful to know.

**Comment by Darkstarinternet** [comment]
I figured it out. I am using a Razer BlackWidow Elite PC keyboard on my Mac Mini, the problem was caused by the keyboard layout I had set in the macOS Settings. I had it set to "British - PC", which I have been using for several years with no problems. It seems however that Kitty doesn't like that particular keyboard layout when using the ` and \ characters which it reverses. I have now switched to a layout called "British (PC 105 alt)" and everything works fine, including the Textual command palette. Thanks again for your help.
